package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-cli-rollbar2",
-  "version": "0.6.1",
+  "version": "0.8.0",
```

Comment:
probably should do something like this, though

Reply:
I'm most likely going to be switching back to yarn, fwiw.